mtp_api/apps/mtp_auth/serializers.py

```diff
@@ -43,6 +43,7 @@ class Meta:
 
     def create(self, validated_data):
         user = self.context['request'].user
+        user.flags.create(name='provided-job-information')
```

Comment:
yup, i'm cool with this happening here. might be nice to have the flag in `mtp_auth.constants` to be able to refer to it consistently

Reply:
Another thought this morning, this does mean we have the flag declared in 2 different apps (a constant here and a constant in the __init__.py of noms-ops app).

Is it actually less fragile to update the flag from noms-ops and just have the constant set in the one app?

Reply:
not sure it makes any real difference… your call :)